pep/app/pods/components/modal-dialogs/user/login/template.hbs

```diff
@@ -41,6 +41,11 @@
                         </a>.
                     </p>
                     --}}
+                    <p class="mb-0 text-muted">
+                        <a href="https://www.psychoanalystdatabase.com/Pages/pg100Home.asp" target="_blank" rel="noopener noreferrer">
```

Comment:
@bakerac4 this should be an ENV var, like `FORGOT_PW_URL`. and probably one we should leave blank in the stage/prod .env files, so that @lhorne-gavant can populate it in the deploy scripts.

Reply:
I was under the assumption that it wouldn't change per environment? Are you sure it will?

Reply:
it might not be yet, but i could easily see that happening with all the PaDS changes. i've just tried to keep all external URLs environment based so they can easily be changed if needed w/o app changes, as even if its the same across envs, it could easily change as their systems change.

If we really don't want this as an env var yet tho, i think it'd be nice to at least have it be a constant (either in the component file or a constant file).